simple_history/management/commands/_populate_utils.py

```diff
@@ -15,16 +16,57 @@ def get_history_model_for_model(model):
     return getattr(model, manager_name).model
 
 
-def bulk_history_create(model, history_model, batch_size):
-    """Save a copy of all instances to the historical model."""
-    historical_instances = [
-        history_model(
-            history_date=getattr(instance, '_history_date', now()),
-            history_user=getattr(instance, '_history_user', None),
-            **{
-                field.attname: getattr(instance, field.attname)
-                for field in instance._meta.fields
-            }
-        ) for instance in model.objects.all()]
-    history_model.objects.bulk_create(historical_instances,
-                                      batch_size=batch_size)
+def bulk_history_create(
+        log_file, model, history_model, batch_size):
+    """Save a copy of all instances to the historical model.
+    :param log_file: file object used to log to, can pass sys.stdout for
```

Comment:
Why the addition of a log_file? Don't love changing this API to include this. Don't think we log anywhere else. Also planning to use this outside of just management commands so I'd avoid this logging here.

Reply:
in management commands you traditionally log to sys.stdout and I'd prefer to have logging here that works in the management command. would logger.info still show up in the management command?

Reply:
Fine with making logging an optional parameter here (which can be triggered by a management command flag). Does that work for you?